### PR TITLE
Jupyter completion

### PR DESCRIPTION
**Pull request by gforsyth**
Updates to the jupyter kernel to work with the new completion framework.

### CODE REVIEW

File: xonsh/jupyter_kernel.py [hunk 1]
```diff
@@ -96,8 +101,13 @@ def _respond_in_chunks(self, name, s, chunksize=1024):
...
         """Get completions."""
         shell = builtins.__xonsh_shell__
-        comps, beg, end = shell.completer.find_and_complete(code, pos, shell.ctx)
-        message = {'matches': comps, 'cursor_start': beg, 'cursor_end': end+1,
+        line = code.split('\n')[-1]
+        prefix = line.split(' ')[-1]
+        endidx = pos
+        begidx = pos - len(prefix)
+        rtn, _ = self.completer.complete(prefix, line, begidx,
+                                                endidx, shell.ctx)
```
Comment (scopatz) [inline xonsh/jupyter_kernel.py]: I think flake8 is complaining about the indentation of this line

**Comment by scopatz** [comment]
Thanks @gforsyth! This seems to fix it locally for me.

**Comment by gforsyth** [comment]
This is ready for another look, @scopatz.  Friggin flake8 trying to ruin my day

**Comment by laerus** [comment]
i thought we were ignoring these pep8 errors :(

**Pull request merged**